spec/abci/abci++_basic_concepts.md

```diff
@@ -375,17 +374,21 @@ the chain once they are verified by a subset of validators. These evidences will
 passed on to the Application through ABCI. It is the responsibility of the
 Application to handle evidence of misbehavior and exercise punishment.
 
-There are two forms of evidence: Duplicate Vote and Light Client Attack. More
+There are two forms of evidence of misbehavior: Duplicate Vote and Light Client Attack. More
 information can be found in either [data structures](../core/data_structures.md)
 or [accountability](../light-client/accountability/).
 
-EvidenceType has the following protobuf format:
+`MisbehaviorType` has the following protobuf format:
```

Comment:
According with the changelog, it was renamed here: https://github.com/tendermint/tendermint/pull/8216

Reply:
For some reason, `EvidenceType` is still present under `api/cometbft/abci/v1beta1`.

Reply:
`v0.37.x` uses `v1beta2` which has the `MisbehaviorType`, for reference (table) https://github.com/cometbft/cometbft/tree/main/proto